packages/components/psammead-consent-banner/src/index.jsx

```diff
@@ -22,6 +22,11 @@ import {
 } from '@bbc/gel-foundations/spacings';
 import { getSansRegular } from '@bbc/psammead-styles/font-styles';
 
+const ltrRtl = (ltrValue, rtlValue) =>
```

Comment:
Currently, this `ltrRtl` method is only required within this component. When it is needed in more than one component, we can pull it out into a shared utility package within Psammead.